erpnext/accounts/doctype/purchase_invoice/purchase_invoice.json

```diff
@@ -1,1586 +1,1525 @@
-   "fieldtype": "Column Break"
-  },
-  {
-   "fieldname": "column_break_58",
-   "fieldtype": "Column Break"
-  },
-  {
-   "fieldname": "company_shipping_address_section",
-   "fieldtype": "Section Break",
-   "label": "Company Shipping Address"
-  },
-  {
-   "fieldname": "column_break_126",
-   "fieldtype": "Column Break"
-  },
-  {
-   "fieldname": "company_billing_address_section",
-   "fieldtype": "Section Break",
-   "label": "Company Billing Address"
-  },
-  {
-   "fieldname": "column_break_130",
-   "fieldtype": "Column Break"
-  },
-  {
-   "collapsible": 1,
-   "fieldname": "status_section",
-   "fieldtype": "Section Break",
-   "label": "Status"
-  },
-  {
-   "fieldname": "column_break_177",
-   "fieldtype": "Column Break"
-  },
-  {
-   "collapsible": 1,
-   "fieldname": "additional_info_section",
-   "fieldtype": "Section Break",
-   "label": "Additional Info",
-   "oldfieldtype": "Section Break",
-   "options": "fa fa-file-text",
-   "print_hide": 1
-  }
- ],
- "icon": "fa fa-file-text",
- "idx": 204,
- "is_submittable": 1,
- "links": [],
- "modified": "2022-10-11 13:04:44.304389",
- "modified_by": "Administrator",
- "module": "Accounts",
- "name": "Purchase Invoice",
- "name_case": "Title Case",
- "naming_rule": "By \"Naming Series\" field",
- "owner": "Administrator",
- "permissions": [
-  {
-   "amend": 1,
-   "cancel": 1,
-   "create": 1,
-   "email": 1,
-   "print": 1,
-   "read": 1,
-   "report": 1,
-   "role": "Accounts User",
-   "share": 1,
-   "submit": 1,
-   "write": 1
-  },
-  {
-   "email": 1,
-   "print": 1,
-   "read": 1,
-   "report": 1,
-   "role": "Purchase User"
-  },
-  {
-   "amend": 1,
-   "cancel": 1,
-   "create": 1,
-   "delete": 1,
-   "email": 1,
-   "print": 1,
-   "read": 1,
-   "report": 1,
-   "role": "Accounts Manager",
-   "share": 1,
-   "submit": 1,
-   "write": 1
-  },
-  {
-   "email": 1,
-   "print": 1,
-   "read": 1,
-   "report": 1,
-   "role": "Auditor"
-  },
-  {
-   "permlevel": 1,
-   "read": 1,
-   "role": "Accounts Manager",
-   "write": 1
-  }
- ],
- "search_fields": "posting_date, supplier, bill_no, base_grand_total, outstanding_amount",
- "show_name_in_global_search": 1,
- "sort_field": "modified",
- "sort_order": "DESC",
- "states": [],
- "timeline_field": "supplier",
- "title_field": "title",
- "track_changes": 1
-}
\ No newline at end of file
+  "actions": [],
+  "allow_import": 1,
+  "autoname": "naming_series:",
+  "creation": "2013-05-21 16:16:39",
+  "doctype": "DocType",
+  "document_type": "Document",
+  "engine": "InnoDB",
+  "field_order": [
+   "title",
+   "naming_series",
+   "supplier",
+   "supplier_name",
+   "tax_id",
+   "due_date",
+   "tax_withholding_category",
+   "column_break1",
+   "company",
+   "posting_date",
+   "posting_time",
+   "set_posting_time",
+   "is_paid",
+   "is_return",
+   "apply_tds",
+   "amended_from",
+   "accounting_dimensions_section",
+   "cost_center",
+   "dimension_col_break",
+   "project",
+   "supplier_invoice_details",
```

Comment:
All the changes made here in supplier invoice seems to be lost, may the conflicts weren't resolved properly while updating